sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -1462,17 +1558,20 @@ describe("LocalWorkspace", () => {
         };
         const projectName = "inline_node";
         const stackName = fullyQualifiedStackName(getTestOrg(), projectName, `int_test${getTestSuffix()}`);
-        const stack = await LocalWorkspace.createStack({ stackName, projectName, program });
+        const stack = await LocalWorkspace.createStack(
+            { stackName, projectName, program },
+            withTestBackend({}, "inline_node"),
+        );
 
-        new Promise((f) => setTimeout(f, 1000)).then(() => controller.abort());
+        new Promise((f) => setTimeout(f, 500)).then(() => controller.abort());
         try {
             // pulumi preview
             const previewRes = await stack.preview({
                 signal: controller.signal,
             });
             assert.fail("expected canceled preview to throw");
         } catch (err) {
-            assert.match(err.toString(), /stderr: Command was killed with SIGINT/);
+            assert.match(err.toString(), /stderr: Command was killed with SIGINT|error: preview canceled/);
```

Comment:
Was this flaky?

Reply:
Yup, depending on when the signal is sent we get different error messages here.